gateway/scim.py

```diff
@@ -259,14 +249,17 @@ def delete_device(entry_id):
     entry = session.get(Device,entry_id)
     if not entry:
         return blow_an_error("Device not found",404)
-    if entry.ethermab_extension:
-        ethermab_delete_device(entry.ethermab_extension)
 
-    for cls in [ BleExtension, EtherMABExtension, FDOExtension ]:
+    for cls in [ BleExtension ]:
```

Comment:
Why are we iterating over 1 element?

Reply:
That's just leftovers before I do BLE, which requires its own PR because it's quite complicated.